src/structures/invite.ts

```diff
@@ -30,6 +52,12 @@ export class Invite extends Base {
     this.targetUserType = data.target_user_type
     this.approximateMemberCount = data.approximate_member_count
     this.approximatePresenceCount = data.approximate_presence_count
+
+    this.uses = (data as any).uses
```

Comment:
Later we should make payload for this too

Reply:
Yea, `interface InviteMetadataPayload extends InvitePayload` 🤔